### DISCUSSION TITLE
Best Solution for Take Control

### DISCUSSION THREAD

**Discussion by bloesss** [amidaware/tacticalrmm#1617] [Q&A]
Hi,

we have now successfully installed and used TRMM, but whitout Meshcentral Integration.
As we already use Meshcentral and did not get the connection to work - everytime I try to connect, the Meshcentral Service restarts.

But that might not be the point, as we already have Meshcentral running and it is OK running it outside of TRMM.
Meshcentral has some missing Features like a Smartphone App (that can simulate a mouse as the Web Version can't), slow Video and miserable Scaling Features, we could use other (free) Teamviewer Alternatives in TRMM.

What other System would you prefer as a second Teamviewer Alternative?

**Rustdesk:**
We already tried the integration. Some Work, some not. Don't know why. Different clients behind the same Firewall. But the Scaling works, the Connection is fast. I have a Smartphone App, but can't run it from the TRMM Website on the iPhone (the Handler is missing).

**VNC**:
Did anyone got VNC to work? Would be a great Solution, as there are tons of Clients. The worst about the VNC Client is the missing "2 Monitor" Feature, as they both are always displayed as one Monitor.

**Remotely:**
Is there a Remotely Addin?

What Else?

**Comment by dinger1986** [comment]
How many devices do you have?

For VNC how would that work for mobile?

For me RustDesk Pro is the best way, save the passwords and devices to the addressbook and it syncs to my other devices

**Reply by bloesss** [reply]
Did Not notice that the pro Version is already There. Can you use the Custom url in trmm with Rustdesk pro?

**Reply by dinger1986** [reply]
you mean the url handler?

**Reply by bloesss** [reply]
Yes. And one More Thing: can rustdesk use default Ports like 80,443,53 instead of 11115… ?

**Reply by dinger1986** [reply]
would need to find 6 TCP ports and 1 UDP port, really the answer is no and I dont think they will change it for a while

**Reply by bloesss** [reply]
And what about the URL Handler?

**Reply by dinger1986** [reply]
Works fine, it's the same client just a different server

**Reply by bloesss** [reply]
That Sounds cool i will Try. As you do Not Need the Adress Book and Group Management in Rustdesk when using it in trmm, the Smartphone App with an Adress Book Could Make the pro Version a Deal.

still need meshcentral for Remote Terminal and tunneling, maybe rport is an Alternative
